### PR TITLE
Support release/support branches in auto-pr

### PR DESCRIPTION
**Pull request by komamitsu**
## Description

We've already installed `auto-pr.yaml` which automatically creates PRs based on specified Projects. But it only supports the default branches (`main`/`master`.) We'll need to manage the backports from branch `3` to branch `3.x` after releasing version 4. `auto-pr` needs to support the release/support branches not only the default branches.

## Related issues and/or PRs

N/A

## Changes made

Add regex pattern for the release/support branches to `pull_request.branches` in `auto-pr.yaml`

## Checklist

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes.
- [x] Any remaining open issues linked to this PR are documented and up-to-date (Jira, GitHub, etc.).
- [ ] Tests (unit, integration, etc.) have been added for the changes.
- [x] My changes generate no new warnings.
- [x] Any dependent changes in other PRs have been merged and published.

## Additional notes (optional)

- This change isn't tested, but it doesn't affect current operations.
- I'll backport this change to other ScalarDB projects once confirming it works.

## Release notes

N/A

### CODE REVIEW

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Comment by brfrn169** [comment]
@feeblefakie @komamitsu @Torch3333 Ah it looks like when we add `[skip ci]` to the last commit, we can't merge the PR because the status check (CI) is required in this repository. We could modify the setting to address this issue, but I feel that the status check is important and we shouldn't disable it. What do you think?

**Comment by komamitsu** [comment]
Oh, interesting.

I think we can push an empty commit after a PR is approved as a workaround? I'll do it this time.

**Comment by Torch3333** [comment]
I also think the status check is essential and should not be disabled. 
I did not know we could push an empty commit 😮

**Comment by komamitsu** [comment]
@Torch3333 Yeah, you can do that with `--allow-empty` 😸

**Comment by brfrn169** [comment]
@komamitsu @Torch3333 Thank you! So let's do the workaround @komamitsu suggested for now.

**Pull request merged**
